slack_sdk/socket_mode/async_client.py

```diff
@@ -72,12 +73,16 @@ async def disconnect(self):
     async def connect_to_new_endpoint(self, force: bool = False):
         try:
             await self.connect_operation_lock.acquire()
+            if self.trace_enabled:
+                self.logger.debug("For reconnection, the connect_operation_lock was acquired")
```

Comment:
Is this called in other cases than a reconnect? If so, suggest simpler `"Acquired connect_operation_lock"`